nixos/modules/programs/zsh/zsh.nix

```diff
@@ -89,8 +89,8 @@ in
         description = ''
           Enable zsh-autosuggestions
         '';
+        type = types.bool;
```

Comment:
please make this a separate commit.

Reply:
Thanks for the review, Mic92. I've just done that.